### ISSUE TITLE
App crashing

### ISSUE THREAD

**Issue by chaosLegacy** [spencercarli/react-native-splash-screen-demo#3]
Hello Spencer,
Awesome tutorial, I'm new to react-native also in android, I have an issue with the code you provided I followed each step of the tutorial but my app keep crashing, I put the same dimension as asked for the splash screen 2048 X 2048 in Ape Tools, I've notice that the size of my image in hdpi is much bigger comparing with the default android logo in mipmap-hdpi, why I said that, is because when I change the bitmap source in splash_screen.xml from `android:src="@drawable/screen"` to `android:src="@mipmap/ic_launcher"`  the app is not crashing anymore but the splashscreen is not visible so I went to AndroidManifest.xml and I put back the the original name of the main activity from `android:name=".SplashActivity"` to `android:name=".MainActivity"` and for the theme i leave it `android:theme="@style/SplashTheme"` it worked.

![screenshot_1497172763](https://user-images.githubusercontent.com/11313734/27009810-28832f48-4e87-11e7-9469-2abf0a8c6c94.png)

So basically I have two questions if you may :
1- Is changing the the theme of the MainActivity could Impact at the style of the whole app ?
2- What can I do if i want to work with my own splash screen ? because till now only the android default mipmap are working.

thank you again

**Comment by spencercarli** [comment]
Seems something may have been messed up when setting things up. Changing the main activity will change your app but just in the launch phase, as soon as the app has launched it will go back to using hte MainActivity.

I would suggest giving it another shot. Good luck!

**Comment by chaosLegacy** [comment]
Yeah, you are right I managed to fix it, looking good now thanks to you
I'm closing the issue now
thanks again ^^
